.changeset/three-eagles-cheat.md

```diff
@@ -0,0 +1,5 @@
+---
+"@comet/cms-api": patch
```

Comment:
```suggestion
"@comet/cms-api": minor
```

IMO this is a feature, what do you think?

Reply:
as I talked with @nsams he thougt this is a bug, minor is ok for me?

Reply:
Whatever

Reply:
@johnnyomair what do you want? minor or patch?

Reply:
Patch is fine for me. Will change the changeset accordingly.